### PR TITLE
[WIP] feat: #4767 codebuild implementation

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4767

### CODE REVIEW

File: packages/deployment-service-ui/src/components/pages/pipelines/show.tsx [hunk 1]
```diff
@@ -7,40 +7,39 @@ import { Button, Label, Loader, StatusIndicator, Table, Intent } from '@reapit/e
...
-  DeploymentStatus,
   PipelineModelInterface,
   PipelineRunnerModelInterface,
   TaskModelInterface,
 } from '@reapit/foundations-ts-definitions'
 import { Pagination } from 'nestjs-typeorm-paginate'
 import { PipelineTask } from '@/components/task'
 
-const pipelineStatusToIntent = (status: DeploymentStatus): Intent => {
+const pipelineStatusToIntent = (status: string): Intent => {
```
Comment (willmcvay) [inline packages/deployment-service-ui/src/components/pages/pipelines/show.tsx:17]: Why no enum now?

File: packages/deployment-service/serverless.yml [hunk 2]
```diff
@@ -526,27 +530,27 @@ functions:
...
     events:
       - sqs:
-          queueName: TaskRunner
+          queueName: CodebuildDeploy
           arn:
             Fn::GetAtt:
-              - TaskRunner
+              - CodebuildDeploy
               - Arn
+    timeout: 300
```
Comment (willmcvay) [inline packages/deployment-service/serverless.yml:556]: EOF 😛

**Review by willmcvay — APPROVED** [review]
LGTM 👍🏻

File: packages/deployment-service/serverless.yml (hunk 2 above)
Comment (bashleigh) [reply packages/deployment-service/serverless.yml:556]: :O you got me 😂

File: packages/deployment-service-ui/src/components/pages/pipelines/show.tsx (hunk 1 above)
Comment (bashleigh) [reply packages/deployment-service-ui/src/components/pages/pipelines/show.tsx:17]: yeaaaa I don't want to install aws-sdk on the frontend for CodeBuild namespace 🙄

**Pull request merged**